src/cider_nrepl/plugin.clj

```diff
@@ -22,7 +22,8 @@
         clojure-versions    (when-not clojure-excluded?
                               (->> dependencies
                                    (keep (fn [[id version & _]]
-                                           (when (= id 'org.clojure/clojure)
+                                           (when (and (= id 'org.clojure/clojure)
+                                                      (string? version))
```

Comment:
I'd add some comment here explaining the need for the extra check.

Reply:
Good point, I added one.